Automattic/wp-calypso
# Paid Stats: Add Jetpack Stats to plans pages

Related to https://github.com/Automattic/dotcom-forge/issues/4595


> [!CAUTION]
> These updates are not behind a feature flag, so let's wait to merge this until paid-stats has gone live.

## Proposed Changes

* This PR adds "Jetpack Stats" as a feature to the Explorer plan in these plan comparison pages:
  * /plans/:site and /start/plans
  * Explorer checkout page
  * Link-in-bio flow
  * Blog onboarding flow
  * Newsletter flow
  * The comparison table at the bottom of /plans/:site and /start/plans

Copy:
* The proposed feature name is "Jetpack Stats"
* The proposed feature description (when your cursor hovers over the feature name) is "Finesse your scaling up strategy with detailed insights and data."

## Testing Instructions

* Load this PR in localhost or Calypso Live
* View the feature in each plan page above and hover over it to view the description copy. You can find screenshots to compare with in the code comments below.
* Ensure there are no new console errors.

## Pre-merge Checklist
